coverage/src/main/kotlin/org/rust/coverage/GrcovRunner.kt

```diff
@@ -111,7 +111,7 @@ class GrcovRunner : RsDefaultProgramRunnerBase() {
         private fun startCollectingCoverage(workingDirectory: File, environment: ExecutionEnvironment) {
             val project = environment.project
             val runConfiguration = environment.runProfile as? RunConfigurationBase<*> ?: return
-            val runnerSettings = environment.runnerSettings ?: return
+            val runnerSettings = environment.runnerSettings ?: CoverageRunnerData()
```

Comment:
Can we pass this data into execution environment in tests directly instead of this hack here?